packages/neos-ui/src/Containers/LeftSideBar/NodeTree/index.js

```diff
@@ -136,6 +150,7 @@ export default class NodeTree extends PureComponent {
 export const PageTree = connect(state => ({
     rootNode: selectors.CR.Nodes.siteNodeSelector(state),
     focusedNodesContextPaths: selectors.UI.PageTree.getAllFocused(state),
+    getNodeByContextPath: selectors.CR.Nodes.nodeByContextPath(state),
```

Comment:
as far as I see it: this will change with any node modification; thus redrawing the full tree at EVERY modification.